src/context.js

```diff
@@ -0,0 +1,28 @@
+import React, { useContext } from "react";
+import data from "./data";
+
+const AppContext = React.createContext()
+
+const AppProvider = ({ children }) => {
+    const getAllUsers = () => data
```

Comment:
If it's a function, every time you will call function, instead of returning data that we already have